FrEee.sln

```diff
@@ -1,20 +1,20 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28010.2003
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29911.98
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "FrEee.WinForms", "FrEee.WinForms\FrEee.WinForms.csproj", "{0EE6CCB3-0A56-4353-B9D1-6D2D6C5737A3}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FrEee", "FrEee\FrEee.csproj", "{E19A4330-CE19-45D6-BCFC-375E065EF041}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "FrEee", "FrEee\FrEee.csproj", "{E19A4330-CE19-45D6-BCFC-375E065EF041}"
-EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "FrEee.Tests", "FrEee.Tests\FrEee.Tests.csproj", "{16D44CA4-5037-4537-93BF-C18F21F1A335}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FrEee.Tests", "FrEee.Tests\FrEee.Tests.csproj", "{16D44CA4-5037-4537-93BF-C18F21F1A335}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{E6AD02C7-FCE2-45BF-AB0D-8A5848DC4325}"
 	ProjectSection(SolutionItems) = preProject
 		Performance5.psess = Performance5.psess
 		Rebracer.xml = Rebracer.xml
 	EndProjectSection
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FrEee.WinForms", "FrEee.WinForms\FrEee.WinForms.csproj", "{61615A88-CFD2-474E-BE6D-6D0A46AA6E5C}"

```

Comment:
Why does FrEee.WinForms have the same GUID as FrEee.Tests now?

Reply:
Same here - some mix-up when creating projects to port code. Should be fixed now.